### PR TITLE
cmd/snap: expose validate command

### PR DESCRIPTION
**Pull request by MiguelPires**
Expose the `snap validate` command in the help info

### CODE REVIEW

File: cmd/snap/cmd_validate.go [hunk 1]
```diff
@@ -46,27 +46,30 @@ type cmdValidate struct {
 
 var shortValidateHelp = i18n.G("List or apply validation sets")
 var longValidateHelp = i18n.G(`
-The validate command lists or applies validations sets
+The validate command lists or applies validations sets that state which snaps
+are required or permitted to be installed together.
+
+A validation set can be in monitoring mode, in which case its constraints aren't
+enforced, or in enforcing mode, in which case snapd will not allow operations
+which would result in snaps breaking the validation set's constraints.
```
Comment (degville) [inline cmd/snap/cmd_validate.go:54]:
```suggestion
The validate command lists or applies validation sets that state which snaps
are required or permitted to be installed together.

A validation set can either be in monitoring mode, in which case its constraints
aren't enforced, or in enforcing mode, in which case snapd will not allow
operations which would result in snaps breaking the validation set's constraints.
```

**Review by degville — APPROVED** [review]
This all looks good to me - makes sense. Thanks for adding the extra context.

**Review by pedronis — APPROVED** [review]
thank you, one suggestion

**Review by Meulengracht — APPROVED** [review]
LGTM

**Pull request merged**